src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/settings/CanonicalTag.js

```diff
@@ -96,13 +96,20 @@ export const CanonicalTag = memo(function CanonicalTag(props) {
                 Only list comma-separated parameter names. Do not include
                 values, ampersands, equals, or spaces.
               </small>
-              <Input
+              <TextField
                 type="text"
                 name="canonicalQueryParamWhitelist"
                 value={whitelist}
-                className={styles.Input}
                 onChange={handleWhitelist}
                 placeholder="page,category"
+                size="small"
+                variant="outlined"
+                color="primary"
+                sx={{
+                  backgroundColor: "primary.contrastText",
+                  mt: "8px",
```

Comment:
Use the system units. mt: 1

Reply:
Are `mt` units a material thing? Not seeing them here https://www.w3.org/Style/Examples/007/units.en.html